mdaviz/mda_folder.py

```diff
@@ -76,6 +76,16 @@ def doFileSelected(self, index):
             self.select_fields_tableview.selected.connect(self.doPlot)
             self.setStatus(f"Selected file: {self.mdaFileList()[index.row()]}")
 
+            if self.mda_file_visualization.isPlotBlankQt():  # TODO what about MPL?
```

Comment:
What about this `TODO` marker?

Reply:
At the moment both MPL and Qt are handle identically (clear clears both graphs, add adds to both graph etc...) so if MPL is blank, Qt is blank.